library/test/test_api_create_device_android.cpp

```diff
@@ -24,6 +25,233 @@
 #include <vulkan/vulkan_android.h>
 #include <vulkan/debug/vulkan_profiles.hpp>
 
+#if defined(ANDROID)
```

Comment:
Similar/identical boilerplate will have to be added to every test `.cpp` file,  yes?  Both here and in other projects that try to use the CMake Android support...

I'm wondering whether there's a way to package this in such a way as to make it easier to use for many tests and in many projects... but it's late and I'm stupid and I'm not coming up with anything workable...